example/samples/install_esx_payload_full.json

```diff
@@ -0,0 +1,75 @@
+{
+    "options": {
+        "defaults": {
+            "version": "6.0",
+            "repo": "http://172.31.128.1:9080/esxi/6.0",
+            "rootPassword": "RackHDRocks!",
+            "hostname": "rackhd-node",
+            "domain": "abc.com",
```

Comment:
examples should use example.com or another domain reserved for examples in RFC 6761.  Don't use a domain that's in existence and is unrelated to RackHD.